qiskit_experiments/calibration/calibrations_manager.py

```diff
@@ -0,0 +1,742 @@
+            qubits: The qubits for which to register the parameter.
+        """
+        if parameter not in self._hash_map:
+            self._hash_map[parameter] = self._parameter_counter
+            self._parameter_counter += 1
+
+        sched_name = schedule.name if schedule else None
+        key = ParameterKey(sched_name, parameter.name, qubits)
+        self._parameter_map[key] = parameter
+        self._parameter_map_r[parameter].add(key)
+
+    @property
+    def parameters(self) -> Dict[Parameter, Set[ParameterKey]]:
+        """
+        Returns a dictionary mapping parameters managed by the calibrations to the schedules and
+        qubits and parameter names using the parameters. The values of the dict are sets containing
+        the parameter keys. Parameters that are not attached to a schedule will have None in place
+        of a schedule name.
+        """
+        return self._parameter_map_r
+
+    def calibration_parameter(
+        self, parameter_name: str, qubits: Tuple[int, ...] = None, schedule_name: str = None
+    ) -> Parameter:
+        """
+        Returns a Parameter object given the triplet parameter_name, qubits and schedule_name
+        which uniquely determine the context of a parameter.
+
+        Args:
+            parameter_name: Name of the parameter to get.
+            qubits: The qubits to which this parameter belongs. If qubits is None then
+                the default scope is assumed.
+            schedule_name: The name of the schedule to which this parameter belongs. A
+                parameter may not belong to a schedule in which case None is accepted.
+
+        Returns:
+             calibration parameter: The parameter that corresponds to the given arguments.
+
+        Raises:
+            CalibrationError: If the desired parameter is not found.
+        """
+        # 1) Check for qubit specific parameters.
+        if (schedule_name, parameter_name, qubits) in self._parameter_map:
+            return self._parameter_map[(schedule_name, parameter_name, qubits)]
+
+        # 2) Check for default parameters.
+        elif (schedule_name, parameter_name, None) in self._parameter_map:
+            return self._parameter_map[(schedule_name, parameter_name, None)]
+        else:
+            raise CalibrationError(
+                f"No parameter for {parameter_name} and schedule {schedule_name} "
+                f"and qubits {qubits}. No default value exists."
+            )
+
+    def add_parameter_value(
+        self,
+        value: Union[int, float, complex, ParameterValue],
+        param: Union[Parameter, str],
+        qubits: Tuple[int, ...] = None,
+        schedule: Union[Schedule, str] = None,
+    ):
+        """
+        Add a parameter value to the stored parameters. This parameter value may be
+        applied to several channels, for instance, all DRAG pulses may have the same
+        standard deviation.
```

Comment:
For docstrings, I usually put a short description on a single line, then an empty line, and then further description. The motivation for this format is that some tools display just the first line of the docstring (like editor pop-ups), so having a concise description looks best.

```suggestion
        Add a parameter value to the stored parameters.
        
        This parameter value may be applied to several channels, for instance, all 
        DRAG pulses may have the same standard deviation.
```

Reply:
Thanks. I did not know that!